tools/pckgen

```diff
@@ -349,6 +363,24 @@ deb_control_files_prepare() {
         sed -i "s/<PACKAGE_MAINTAINER>/$PACKAGE_MAINTAINER/g" "$file"
         sed -i "s/<PACKAGE_DATE>/$PACKAGE_DATE/g" "$file"
     done
+
+    if [ "$DEBUG" ]; then
+        echo "---   Debug info will be included and debug packages created as well"
+        local deb_rules_file="$DEB_SOURCES_DIR/debian/rules"
+
+        # We need to find the line responsible for compiling process
+        # (to add proper flag to it), and also the line which strips
+        # debug symbols (to force debug package creation).
+        local make_pattern='make -C casadm$'
+        local strip_pattern='dh_strip.*--no-ddebs'
+
+        if ! grep "$make_pattern" "$deb_rules_file" &>/dev/null \
+                || ! grep "$strip_pattern" "$deb_rules_file" &>/dev/null; then
+            error "couldn't find proper rules for debug build in file $deb_rules_file"
+        fi
+        sed -i "/$make_pattern/ s/$/ DEBUG_PACKAGE=1/g" "$deb_rules_file"
+        sed -i "/$strip_pattern/ s/--no-ddebs/--ddebs/g" "$deb_rules_file"
```

Comment:
The same comment here. Try to use templating instead.

Reply:
done